api/environments.md

```diff
@@ -0,0 +1,36 @@
+---
+description: 'Tradecloud API environments available'
+---
+
+# Environments
+
+Tradecloud has two physical separated infrastructure:
+
+- test infrastructure
+- production infrastructure
+
+Tradecloud has three environments:
+
+- [Test environments](#test-environments)
+- [Acceptance test environment](#acceptance-test-environment)
+- [Production environment](#production-environment)
+
+## Test environments
+
+There are multiple test environments which are deployed to develop and test a feature, have a short dynamic life cycle and have NO availability SLA. The test environments are deployed on the test infrastructure which has a lower availability.
+
+Available [test environments API page](https://api.test.tradecloud1.com)
```

Comment:
Because the Damen environment is a test environment and it does not have its own service/status page like https://api.damen.test.tradecloud1.com. Damen en master are published first in https://api.test.tradecloud1.com

Reply:
Ok, makes sense. It would be nice to not have this exposed at a later stage, when eg. `accp` is used for everyone to test.